<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo Metode QMessageBox</string>
  </property>
  <widget class="QPushButton" name="aboutButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
  <widget class="QPushButton" name="criticalButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Critical</string>
   </property>
  </widget>
  <widget class="QPushButton" name="informationButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="questionButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Question</string>
   </property>
  </widget>
  <widget class="QPushButton" name="warningButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Warning</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
